data/hai/hai reveal 0 prologue.txt

```diff
@@ -0,0 +1,734 @@
+			`	"No," Xilin says while looking away. Something else has caught his attention. "But they might be."`
+			`	You look up to see a beat-up Berserker that looks like it can barely fly landing in the spaceport. "You're free to go," Xilin says to the Star Queen captain.`
+			`	You approach the Berserker as it finishes landing. The captain of the ship exits looking far calmer than the condition of the Berserker would suggest its captain should be. "Uh, hi," he says akwardly. "Can I help you guys?"`
+			choice
+				`	"What happened to your ship?"`
+				`	"Did you get attacked by pirates on the way here."`
+			`	The captain looks back at the poor state of his interceptor. "Oh, that. Yeah, I ran into a couple pirates on the way here. Nothing I haven't dealt with before, though."`
+			`	"So you've been to Hai space before?" Xilin asks.`
+			`	"Sure. Plenty of times," he responds.`
+			`	Xilin looks over the interceptor with a quizzical expression. "May I see your pilot's license?"`
+			`	"Yeah, no problem," the captain says, as he quickly pulls a license out of his pocket that looks almost as beat up as the interceptor. Xilin jots down the details and hands the license back to the captain. "Thank you. And maybe tag along with a merchant fleet on your way back home."`
+			`	"That sounds like a good idea," the captain says before walking off into the spaceport.`
+			`	You continue to question more captains in the spaceport over the course of the day with Xilin and Sayari. Some already knew of the Hai before, but many are new arrivals who heard of the way to the Hai through whispers in spaceport bars, or were sold coordinates by knowledgeable captains. Everyone seems peaceful enough in interacting with the Hai, but clearly the word is getting out more as time goes on.`
+	on visit
+		dialog phrase "generic missing stopover or passengers"
+	on complete
+		conversation
+			`You return to the briefing room at the secure port with Xilin and Sayari, and Xilin begins frantically copying notes into a terminal. He then pauses while staring at something on the screen.`
+			`	"I'm running each of the pilot's licenses and ship registrations through my copy of the Republic's pilot database. They've all come up as valid and with mostly clean records. All except for one." He turns the terminal screen toward you and Sayari, displaying an inputted license ID that the system says does not exist.`
+			`	"This is from that Berserker pilot. What's more is that if I run the ID of the ship, it's registered as stolen."`
+			`	Sayari gives a look of concern. "Are you suggesting that this captain was a criminal?"`
+			`	"Highly likely," Xilin responds. "A pirate with a stolen ship and a counterfeit license."`
+			`	"I will inform our spaceport authorities to look out for this captain and his ship," Sayari says. "It may also be beneficial if you were to make this database available to us, so that we may catch other potential criminal humans."`
```

Comment:
I doubt the Hai prosecute humans for crimes not in Hai space. For individuals below a certain threat level, and without an existing arrangement with some Hai entity, it's likely they just monitor them rather than taking any action.

Reply:
```suggestion
			`	"Highly likely," Xilin responds. "A pirate with a stolen ship and a counterfeit license."`
			`	"I will inform our spaceport authorities to look out for this captain and his ship," Sayari says. "It may also be beneficial if you were to make this database available to us, so that we may monitor other potential criminal humans."`
```